src/test/java/io/cdap/plugin/salesforce/parser/SalesforceQueryParserTest.java

```diff
@@ -266,4 +266,10 @@ public void testSubQuery() {
       Collections.singletonList(new SObjectDescriptor("Contacts", fields)),
       result.getChildSObjects());
   }
+
+  @Test
+  public void testParseFailure() {
```

Comment:
Rename to something like testKeywordInQuery

Reply:
Method renamed from testParseFailure to testKeywordInQuery